AutoCollection/HttpRequests.ts

```diff
@@ -44,7 +45,7 @@ class AutoCollectHttpRequests {
     }
 
     public useAutoCorrelation(isEnabled:boolean, forceClsHooked?:boolean) {
-        if (isEnabled && !this._isAutoCorrelating) {
+        if (isEnabled && !this._isAutoCorrelating && !process.env[Config.ENV_disableDependencyCorrelation]) {
```

Comment:
What's the intention of this env var? To provide a way to change the default setting to disabled? Or to override explicit enabling in code?

I was expecting the former, though this looks like the latter. Totally fine if intentional.

Reply:
Intentionally the latter. I'll look into whether or not or not this is the behavior of the other similar env vars